docs/service-brokers/helm-broker-service-classes/gcp-broker/docs/overview.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Overview
+type: Overview
+---
+>**NOTE:** You must create a Secret with provider credentials in every Namespace where you provision the GCP Service Broker class. You can provision only one instance of the GCP Service Broker in each Namespace.
```

Comment:
```suggestion
>**NOTE:** You can provision only one instance of the GCP Service Broker in each Namespace. In every Namespace, you must create a Secret with provider credentials. Read the following section to learn how.
```

Reply:
Let's discuss with PK (the order os sentences)